plugin/output/file/timestamp/dates.go

```diff
@@ -0,0 +1,44 @@
+package timestamp
```

Comment:
Here I suggest not to create a separate package for this. Just add this file to `plugin/output/file/`